### DISCUSSION TITLE
Is it possible to configure compress level?

### DISCUSSION THREAD

**Discussion by filipegarrido15** [sidorares/node-mysql2#2186] [Q&A]
I'm analyzing ways to optimize query performance and I'm studying data compression in MySQL. In the documentation it says that the client defines the compression level, so I wanted to know if it's possible to configure the compression level when creating a new connection.

**Comment by sidorares** [comment]
I don't think there is a way to set the level, but the client may choose to use zstd compression is the server supports it. Its currently not implemented here yet though ( see the tracking issue - https://github.com/sidorares/node-mysql2/issues/1815 )

Compression documentation: https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_basic_compression.html
